app/index.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+var validUrl = require('valid-url');
+var urlModel = require('./model/url');
+var config = require('./../config');
+
+var responseCodes = {
+    httpOK: 200,
+    httpCreated: 201,
+    httpBadRequest: 400,
+    httpNotFound: 404,
+    httpError: 500,
+};
+
+function cropLastSlash(str) {
+    if(str.substr(-1) === '/') {
+        return str.substr(0, str.length - 1);
+    }
+    return str;
+};
+
+module.exports = function(app) {
+    app.listen(config.port, function(){
+        console.log('Server is listening on port 8000');
```

Comment:
const port = config.port;

console.log(`Server is listening on port ${port}`);